migration/sql-files/077-work_item_number_sequence.sql

```diff
@@ -0,0 +1,5 @@
+-- modify the `work_item_number_sequences` to include an independant ID column for the primary key
+alter table work_item_number_sequences drop constraint work_item_number_sequences_pkey;
+alter table work_item_number_sequences add column  id uuid primary key DEFAULT uuid_generate_v4() NOT NULL;
```

Comment:
You can add a test for checking the existence of `id` column in `migration/migration_blackbox_test.go`

Reply:
yes, you're right